src/index.html

```diff
@@ -2,11 +2,308 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <!-- Card 1 -->
+      <div
+        class="card"
+        data-qa="card">
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="card__review-stars stars">
```

Comment:
```suggestion
          <div class="card__review-stars stars stars--4">
```
The number of active stars should depend on the class with the modifier